src/wallet/web.ts

```diff
@@ -0,0 +1,44 @@
+import { web3, Wallet } from "@project-serum/anchor";
+import { SendOptions } from "../program";
+
+export default class WebWallet implements Wallet {
+  _signTransaction: (transaction: web3.Transaction) => Promise<web3.Transaction>;
+  _signAllTransactions: (transaction: web3.Transaction[]) => Promise<web3.Transaction[]>;
+  sendTransaction: (transaction: web3.Transaction, connection: web3.Connection, options?: SendOptions) => Promise<string>;
+  _publicKey: web3.PublicKey;
+  payer: web3.Keypair;
+
+  static fakeWallet() {
+    return new WebWallet(
+      web3.PublicKey.default,
+      (transaction: web3.Transaction) => {return Promise.resolve(new web3.Transaction())},
+      (transaction: web3.Transaction[]) => {return Promise.resolve([new web3.Transaction()])},
```

Comment:
Can we remove `transaction` property from these fake `_signTransaction` and `_signAllTransactions` functions?
I think it's useless.

```suggestion
      () => Promise.resolve(new web3.Transaction()),
      () => Promise.resolve([new web3.Transaction()]),
```

Reply:
Updated to always use the argument and set a default for the arguments